src/pages/MetricDetail.svelte

```diff
@@ -117,6 +119,25 @@
     {/each}
     ping{metric.send_in_pings.length > 1 ? 's' : ''}.
   </p>
+
+  {#if metric.extra_keys && !isEmpty(metric.extra_keys)}
+    <h2>
+      Extra key{Object.keys(metric.extra_keys).length > 1 ? 's' : ''}
```

Comment:
Looking at your screenshot, I think it makes more sense to just always use the plural. This really is a group of things, and the plural makes that more clear.

```suggestion
      Extra keys
```